src/components/dialog/DetachPermissionFromRoleDialog.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <q-dialog v-model="show">
+    <q-card>
+      <q-card-section class="flex row justify-center">
+        <span class="text-h6 text-center">
+          {{ $t('DetachPermissionFromRoleDialog.text.title',
+                { permission: permission.key, role: role.name }) }}
```

Comment:
{ permission: $t{permission.key}, role: role.name }) }}

just an example but give the description of the permission not the key